### PR TITLE
BLOCKCHAIN-126 - merge LLD staking, nominating and validating to a dedicated Staking page

### PR DESCRIPTION
**Pull request by kacperzuk-neti**
Now our frontend can be used for both nominating and validating. As these share the same stash, stake & bond logic, it makes sense to bring it all to a single place.

This PR handles all the cases:
* starting from scratch to nominating
* starting from scratch to validating
* nominating with stash already created
* validating with stash already created
* switching between validating and nominating

### CODE REVIEW

File: src/api/nodeRpcCall.js [hunk 1]
```diff
@@ -428,7 +428,7 @@ const stakeToPolkaBondAndExtra = async (payload, callback) => {
   const api = await getApi();
   const transferExtrinsic = isUserHavePolkaStake
     ? await api.tx.staking.bondExtra(dollarsToGrains(amount))
-    : await api.tx.staking.bond(walletAddress, dollarsToGrains(amount), 'Staked');
```
Comment (kacperzuk-neti) [inline src/api/nodeRpcCall.js:431]: API changed in Runtime [v11.0.3](https://github.com/liberland/liberland_substrate/releases/tag/v11.0.3).

File: src/components/Modals/ChoseStakeModal.js [hunk 2]
```diff
@@ -40,20 +40,6 @@ function ChoseStakeModal({
           >
             Unpool
           </Button>
-          <Button
```
Comment (kacperzuk-neti) [inline src/components/Modals/ChoseStakeModal.js:43]: Moved to Staking page

File: src/components/Staking/Nominator/ValidatorCard/ValidatorCard.js [hunk 3]
```diff
@@ -12,17 +12,23 @@ function ValidatorCard({
   validatorAddress,
   selectingValidatorsDisabled,
 }) {
+  let checkbox = '';
+  if (nominatedByMe) checkbox = '✅';
+  else if (selectingValidatorsDisabled) checkbox = 'MAX';
```
Comment (kacperzuk-neti) [inline src/components/Staking/Nominator/ValidatorCard/ValidatorCard.js:17]: Making eslint happy (forbids nested ternary operators)

File: src/components/Staking/Nominator/styles.module.scss [hunk 4]
```diff
@@ -9,7 +9,7 @@
   justify-content: center;
   border-radius: 20px;
   //background: $bg3;
-  background: $transparent3;
+  //background: $transparent3;
```
Comment (kacperzuk-neti) [inline src/components/Staking/Nominator/styles.module.scss:12]: @MichalLustykNeti will work out layout & styling in next PR

**Review by MichalLustykNeti — APPROVED** [review]

**Pull request merged**